cibuildwheel/crosslinux.py

```diff
@@ -0,0 +1,536 @@
+import os
+import subprocess
+import sys
+import textwrap
+from pathlib import Path, PurePath
+from typing import Dict, List, NamedTuple, Optional, Set
+
+from .architecture import Architecture
+from .docker_container import DockerContainer
+from .logger import log
+from .typing import PathOrStr
+from .util import (
+    BuildOptions,
+    BuildSelector,
+    NonPlatformWheelError,
+    get_build_verbosity_extra_flags,
+    prepare_command,
+    read_python_configs,
+)
+
+# Docker container which will be used to resove dependencies
+# while crosss compiling the wheels
+native_docker_images = {"aarch64": "quay.io/pypa/manylinux2014_aarch64"}
```

Comment:
This should be pulled from the normal setting for linux aarch64, not hardcoded. I'm assuming this also should be used for tests?